README.md

```diff
@@ -54,7 +54,7 @@ BiocManager::install("miaSim")
 
 miaSim is accompanied by miaSimShiny, which allows users to explore the parameter space of their models in real-time in an intuitive graphical interface. miaSimShiny is available at https://github.com/gaoyu19920914/miaSimShiny.
```

Comment:
You could add the hyperlink directly on the text "miaSimShiny", no need to show the URL explicitly in the text.